src/main/kotlin/com/expedia/graphql/schema/extensions/directiveExtensions.kt

```diff
@@ -0,0 +1,60 @@
+package com.expedia.graphql.schema.extensions
+
+import com.expedia.graphql.annotations.GraphQLDirective
+import com.expedia.graphql.schema.exceptions.CouldNotGetNameOfAnnotationException
+import com.expedia.graphql.schema.generator.SchemaGenerator
+import com.google.common.base.CaseFormat
+import graphql.schema.GraphQLArgument
+import graphql.schema.GraphQLInputType
+import kotlin.reflect.KAnnotatedElement
+
+internal fun KAnnotatedElement.directives(generator: SchemaGenerator) =
```

Comment:
We only need to expose a single function to get directives on any element that has an annotation